app/controllers/users_controller.rb

```diff
@@ -10,11 +10,10 @@ def create
       flash[:notice] = "You are now registered and logged in."
       redirect_to '/profile'
     else
-      flash[:notice] = @user.errors.full_messages.to_sentence
+      flash[:notice] = @user.errors.full_messages.uniq.to_sentence
```

Comment:
Added that uniq method there because the password confirmation gave me two repeated errors. Seems like a bit of a bandaid so if anyone has ideas as to why I was getting duplicate errors please let me know!

My best guess is that maybe it's checking the confirmation both ways:
password == password_confirmation
password_confirmation == password

That's just a guess though. Really unsure why it's behaving that way.

Reply:
I think `.uniq` is a smart fix for now. Not sure about the duplicate errors... I'm researching the docs to get a better handle on password confirmation.

Reply:
I saw those duplicate errors and couldn't pinpoint why it was doing that so I'm glad this works.

Reply:
That's interesting.  In an unrelated note, do you think it's because we've named all our flash notices flash[:notice]?  I don't think this suggestion would fix the duplication errors, but I do believe that we wouldn't really know how to pin point it if the flash notice has the same name?  I'm wondering what the pro/cons are for naming all our flash notices this way?

I agree with everyone else, @AngelaGuardia, this seems like a good enough fix for now.